File: models/cabin.js
const mongoose = require('mongoose');
const user = require('./user');

// datanamn för vår Collection med respektive datatyper
const cabinSchema = mongoose.Schema({
    _id: mongoose.Types.ObjectId, // speciell datatyp, mongoose specifik "id"
    owner:  { type: String, required: true },
    adress: { type: String, required: true },
    m2:     { type: Number, required: true },
    sauna:  { type: String, required: true },
    beach:  { type: String, required: true }
});

module.exports = mongoose.model('Cabin', cabinSchema);

File: Models/Cabin.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MyRoomBookingApi.Models
{
    public class Cabin
    {

        public string _id       { get; set; }
        public string owner     { get; set; }
        public string adress    { get; set; }
        public int m2           { get; set; }
        public string sauna     { get; set; }
        public string beach     { get; set; }
        public string ownerId   { get; set; }
    }

    public class RootClass
    {
        public Cabin Cabin { get; set; }
    }
}
